### PR TITLE
Conditionally show call in comment instructions

### PR DESCRIPTION
**Pull request by hancush**
## Overview

This PR hides call in instructions for a subset of committees that do not accept live comment.

### Checklist

- [x] PR has a descriptive enough title to be useful in changelogs

## Testing Instructions

 * Scrape agendas for upcoming events:
```bash
docker-compose run --rm scrapers pupa update lametro bills window=7 --rpm=0
docker-compose run --rm scrapers pupa update lametro events window=7 --rpm=0
```
 * View the 3/24 board meeting and confirm you see call in instructions
 * View the 3/31 Measure R meeting and confirm you do not see call in instructions

Handles #820

### CODE REVIEW

File: lametro/templates/partials/event_header_live_public_comment.html [hunk 1]
```diff
@@ -0,0 +1,122 @@
+<div class="row">
```
Comment (hancush) [inline lametro/templates/partials/event_header_live_public_comment.html:1]: This is copy/pasted exactly from `event.html`.

File: lametro/templates/partials/event_header_no_live_public_comment.html [hunk 2]
```diff
@@ -0,0 +1,54 @@
+<div class="row">
```
Comment (hancush) [inline lametro/templates/partials/event_header_no_live_public_comment.html:1]: This is almost identical to the public comment partial, but omits ecomment and call in instructions. Defining these separately makes the conditional logic easier to follow.

File: lametro/models.py [hunk 3]
```diff
@@ -722,6 +722,17 @@ def ecomment_message(self):
             return self.UPCOMING_ECOMMENT_MESSAGE
 
 
+    @property
+    def accepts_live_comment(self):
+        meetings_without_live_comment = (
```
Comment (fgregg) [inline lametro/models.py:727]: not a requirement but making this a set with

{
           'Measure R Independent Taxpayer Oversight Committee',
            'Measure M Independent Taxpayer Oversight Committee',
            'Independent Citizen’s Advisory and Oversight Committee',
}

make intent slightly clearer.

File: tests/test_events.py [hunk 4]
```diff
@@ -564,3 +564,40 @@ def test_meeting_stream_link_unavailable(mocker):
...
+    test_event = event.build(name=event_name, start_date=in_an_hour)
+    event_document.build(note='Agenda', event_id=test_event.id)
+
+    assert test_event.accepts_live_comment == expected_live_comment_value
+
+    url = reverse('lametro:events', args=[test_event.slug])
+    response = client.get(url)
+    response_content = response.content.decode('utf-8')
+
+    live_comment_signature = (
```
Comment (fgregg) [inline tests/test_events.py:596]: you are testing both a model and a template rending here. please split into two tests. the test for the template just needs a true and false condition.

**Review by fgregg — CHANGES_REQUESTED** [review]
looks good, would be good to split the test in two, since you might want to change template at some point.

File: tests/test_events.py (hunk 4 above)
Comment (hancush) [reply tests/test_events.py:610]: Great call, done in https://github.com/datamade/la-metro-councilmatic/pull/823/commits/8c8383b851a5df352d9c5d361edcc8a5168c9979.

File: lametro/models.py (hunk 3 above)
Comment (hancush) [reply lametro/models.py]: Done in https://github.com/datamade/la-metro-councilmatic/pull/823/commits/8c8383b851a5df352d9c5d361edcc8a5168c9979.

**Comment by hancush** [comment]
Thanks for keeping me honest, @fgregg! Made both requested changes, back over to you.

**Pull request merged**